classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION', '4.0.2' );
```

Comment:
Unless you need a specific feature, I'd use the same logic we use for the minimum WP version-- Current minus 1, so, in this case, 3.9